rtmp_to_hls/README.md

```diff
@@ -0,0 +1,18 @@
+# RtmpToHls
```

Comment:
We should probably mention what it is right around here

Reply:
Also, I think we also need usage instructions, maybe including tutorial on setting up OBS to stream to this server

Reply:
done